lxc/exec_windows.go

```diff
@@ -32,11 +34,24 @@ func (c *execCmd) getTERM() (string, bool) {
 }
 
 func (c *execCmd) controlSocketHandler(control *websocket.Conn) {
-	// TODO: figure out what the equivalent of signal.SIGWINCH is on
-	// windows and use that; for now if you resize your terminal it just
-	// won't work quite correctly.
-	err := c.sendTermSize(control)
-	if err != nil {
-		logger.Debugf("error setting term size %s", err)
+	ch := make(chan os.Signal, 10)
+	signal.Notify(ch, os.Interrupt)
```

Comment:
The code sets up a signal handler for os.Interrupt. If the OS sends os.Interrupt to the lxc process, Go will write that signal to the channel which will then be sent as SIGINT to LXD using the control socket.